src/Command/IndexService/BootstrapCommand.php

```diff
@@ -22,13 +22,19 @@
 use Pimcore\Console\Traits\Timeout;
 use Pimcore\Model\DataObject;
 use Pimcore\Model\DataObject\Listing;
+use Symfony\Component\Console\Attribute\AsCommand;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Input\InputOption;
 use Symfony\Component\Console\Output\OutputInterface;
 
 /**
  * @internal
  */
+#[AsCommand(
+    name: 'ecommerce:index:create-or-update-indices',
```

Comment:
Need to change the docs to use the replacement 
https://github.com/pimcore/ecommerce-framework-bundle/edit/1.0/doc/05_Index_Service/01_Product_Index_Configuration/05_Data_Architecture_and_Indexing_Process.md

About the naming, `indices` is fine, but would be `indexes` (both are grammatically fine) easier to read for non-natives? and could be just avoided because the part `:index:` is already hinting the domain/area of application?

Reply:
Thanks for the suggestion. Yes, we can rename it to 'ecommerce:index:create-or-update'. Should we also consider renaming the options https://github.com/pimcore/ecommerce-framework-bundle/blob/a576c8470d005df9312696a602d7f9a6ded33f46/src/Command/IndexService/BootstrapCommand.php#L65